app/shared/services/DownstreamResponseMappingSupport.scala

```diff
@@ -32,6 +32,10 @@ trait DownstreamResponseMappingSupport {
         logger.warn(s"[${logContext.controllerName}] [${logContext.endpointName}] - No matching stub was found")
         RuleIncorrectGovTestScenarioError
 
+      case "INVALID_CORRELATION_ID" | "INVALID_CORRELATIONID" =>
```

Comment:
Actually one thing about adding this to BSAS is that you'd probably also need to go through all of the service classes, removing the same error mapping. (Not from the tests though). I'd suggest not merging this PR for now (maybe set to Draft, with a comment), as there ought to be more of a review of which "common" errors should be handled in the shared code, rather than just the one error.